plugins/BEdita/Core/src/Model/Action/SignupUserAction.php

```diff
@@ -309,15 +310,15 @@ protected function createUserEntity(array $data, $status, $validate)
             ]);
         }
         $action = new SaveEntityAction(['table' => $this->Users]);
+
         $data['status'] = $status;
+        $entity = $this->Users->newEntity();
+        $entityOptions = compact('validate');
+        if (!empty($data['verified'])) {
+            $entityOptions['accessibleFields'] = ['verified' => true];
+        }
```

Comment:
In this way passing `$data['verified']` the verification will be bypassed. I think it would be better if `$verified` was passed as fourth parameter

```php
protected function createUserEntity(array $data, $status, $validate, $verified = false)
```
In this way we don't break backward compatibility.

Reply:
Using this approach, I'd suggest setting the `verified` field directly without going through `patchEntity`:

```php
if ($verified === true) {
    $entity->set('verified', FrozenTime::now());
}
```

Setting fields directly bypasses validation, and we can avoid messing with accessible fields.

Reply:
ok, I'd use the `$entity->set()` approach with new `$verified` argument